### PR TITLE
Deploy new app version to dev on commit to master

### PR DESCRIPTION
**Pull request by khbnr**
Working pipeline, see https://drone.dev.healthmetrix.com/labres/server/340/1/8

On commit to master, upload built jar to S3, create a new beanstalk application version for it and update the beanstalk environment to use the new application version (all on dev)

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -32,6 +37,48 @@ steps:
     commands:
       - ./gradlew --no-daemon dependencyUpdates
 
+  - name: dev - assume role
```
Comment (0teh) [inline .drone.yml:40]: Should this step also occur on master only?

File: .drone.yml [hunk 2]
```diff
@@ -32,6 +37,48 @@ steps:
...
+        EOF
+
+  - name: dev - release
+    image: amazon/aws-cli
+    environment:
+      BUCKET_NAME:
+        from_secret: dev_bucket
+    commands:
+      - cp -r .aws $HOME/.aws
+      - echo "$${BUCKET_NAME} a${BUCKET_NAME} $BUCKET_NAME"
```
Comment (0teh) [inline .drone.yml:62]: what's this echo doing? also there is a random 'a' in there

Comment (khbnr) [reply .drone.yml]: It was for testing. Tidied it up.

File: .drone.yml (hunk 1 above)
Comment (khbnr) [reply .drone.yml:40]: Indeed. Thanks!

**Pull request merged**